src/components/PlanterQuote.cy.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { mountWithTheme as mount } from '../models/test-utils';
+import PlanterQuote from './PlanterQuote';
+
+beforeEach(() => {
+  cy.intercept('/_next/**', {
+    fixture: 'images/greenway-international.png',
+  });
+  cy.intercept('/_next/image?url=%2Fimages%2Fquote-symbol.svg**', {
+    fixture: 'images/quote-symbol.svg',
+  });
+  cy.viewport(720, 360);
```

Comment:
Create a separate test for the mobile view and set the viewport in each test instead.